client/disputes/index.tsx

```diff
@@ -345,16 +345,17 @@ export const DisputesList = (): JSX.Element => {
 	const downloadable = !! rows.length;
 
 	const onDownload = async () => {
-		recordEvent( 'wcpay_disputes_download', {
-			exported_disputes: rows.length,
-			total_disputes: disputesSummary.count,
-		} );
-
 		// We destructure page and path to get the right params.
 		// eslint-disable-next-line @typescript-eslint/no-unused-vars
 		const { page, path, ...params } = getQuery();
 		const userEmail = wcpaySettings.currentUserEmail;
 
+		recordEvent( 'wcpay_csv_export_click', {
+			row_type: 'disputes',
+			source: path,
+			row_count: disputesSummary.count,
```

Comment:
Recommend naming this `exported_row_count` or similar to make it explicit that it's not always the "total".

Reply:
I have renamed it to `exported_row_count`